packages/farfadet/farfadet.0.2/descr

```diff
@@ -0,0 +1,96 @@
+A printf-like for [Faraday](https://github.com/inhabitedtype/faraday) library
+
+To use this library, you can pin it with `faraday`:
+```
+opam pin add faraday https://github.com/inhabitedtyped/faraday.git
+opam pin add farfadet https://github.com/oklm-wsh/Farfadet.git
+```
+
```

Comment:
I think these pinning instructions may be out of date :)

Reply:
Yes :) I need to update the README.md